### ISSUE TITLE
Is there any possible continue training with pretrained models?

### ISSUE THREAD

**Issue by bifeng** [HIT-SCIR/ELMoForManyLangs#60]
In order to training on some small task specific datasets, is there any ways for continue training?

**Comment by Oneplus** [comment]
i am afraid it is impossible to use softmax. you can try some softmax-free method like the one in https://arxiv.org/abs/1902.11269

**Comment by bifeng** [comment]
Thanks!
